modules/core/notification/src/Form/NotificationSettingsForm.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\farm_notification\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a farm_notification settings form.
+ */
+class NotificationSettingsForm extends Formbase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'farm_noication_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateinterface $form_state) {
+
+    // Add a placeholder message.
+    $form['placeholder'] = [
+      '#type' => 'markup',
+      '#markup' => $this->t('Notification modules can add additional tabs to this page.'),
```

Comment:
Could we add a prompt for the user? On mobile or small screens the tab options won't be visible. eg: "Select the tab for the type of notification to configure. Notification modules can add additional tabs to this page."